### ISSUE TITLE
New and updated entries for Winapp2.ini

### ISSUE THREAD

**Issue by Shakil-Shahadat** [MoscaDotTo/Winapp2#828]
# AIMP [ Updated ]
For reference, I am quoting the `AIMP` entry in `winapp.ini`.
```
[AIMP 3]
ID=2287
LangSecRef=3023
DetectFile=%AppData%\AIMP3
Default=True
FileKey1=%AppData%\AIMP3|AIMP3.bak
FileKey2=%AppData%\AIMP3|AIMP3ate.bak
FileKey3=%AppData%\AIMP3|AIMP3lib.bak
FileKey4=%AppData%\AIMP3|AIMP3ac.bak
```
There are several problems here,
- `AIMP` app is now at version 5 and in `winapp2.ini` it is named as just `AIMP`, not `AIMP 3`.
- `AIMP` directory name in `%AppData%` is now just `AIMP`, not `AIMP3`, thus this won't even detect the app. Although, `winapp2.ini` can properly identify both old and new folder names as it detects `AIMP*`.
- None of the FileKeys exists anymore. Even if they did, they could have been summarised like `*.bak`. The only such file that exists is `AIMP.ini.bak` which it doesn't detect.
In summary, the `AIMP` entry in `winapp.ini` is completely obsolete.

Let me quote the `AIMP` entry in `winapp2.ini` for reference.
```
[AIMP *]
LangSecRef=3023
DetectFile=%ProgramFiles%\AIMP*
FileKey1=%AppData%\AIMP|*.bak
FileKey2=%ProgramFiles%\AIMP*\!Backup|*.*|REMOVESELF
FileKey3=%ProgramFiles%\AIMP*\Data|*.bak|RECURSE
```
The problems here,
- `DetectFile` can be updated to `%AppData%\AIMP*`. This will ensure, the detection of `AIMP` regardless of install location and version.
- Similarly, `FileKey1` should be updated to `%AppData%\AIMP*|*.bak`. This should work in ver 3 to 5, at least.
- Just a note here, there is no `Data` folder in `%ProgramFiles%\AIMP*` in ver 5.

So, the updated entry should be,
```
[AIMP *]
LangSecRef=3023
DetectFile=%AppData%\AIMP*
FileKey1=%AppData%\AIMP*|*.bak
FileKey2=%ProgramFiles%\AIMP*\!Backup|*.*|REMOVESELF
FileKey3=%ProgramFiles%\AIMP*\Data|*.bak|RECURSE
```
# CareUEyes [ New Entry ]
```
[CareUEyes *]
LangSecRef=3021
DetectFile=%AppData%\careueyes
FileKey1=%AppData%\careueyes\log|*.*
```
# DU Meter [ Updated ]
```
[DU Meter *]
LangSecRef=3022
Detect=HKCU\Software\Hagel\DU Meter
FileKey1=%CommonAppData%\Hagel Technologies\DU Meter|*.csv;error.elf
```
Change: `error.elf` is new here. It's just an error log. Also, I don't see any *.csv file there.
# Foxit PDF Editor [ New Entry ]
```
[Foxit PDF Editor *]
LangSecRef=3021
Detect=HKCU\Software\Foxit Software\Foxit PDF Editor 11.0
FileKey1=%AppData%\Foxit Software\Foxit PDF Editor\Cache|*.db
FileKey2=%AppData%\Foxit Software\Foxit PDF Editor\cacheData|*.dat
FileKey3=%AppData%\Foxit Software\Foxit PDF Editor\FormFiller|AutoComplete.ds
RegKey1=HKCU\Software\Foxit Software\Foxit PDF Editor 11.0\CommentPanel\Filter
RegKey2=HKCU\Software\Foxit Software\Foxit PDF Editor 11.0\MRU\File MRU
RegKey3=HKCU\Software\Foxit Software\Foxit PDF Editor 11.0\MRU\Place MRU
RegKey4=HKCU\Software\Foxit Software\Foxit PDF Editor 11.0\Preferences\History
```
Note, `Foxit PhantomPDF` is renamed to `Foxit PDF Editor`. This version is somewhat different, that's why I made this new entry.

**Comment by APMichael** [comment]
The FileKeys can be merged:
> ```
> [DU Meter *]
> LangSecRef=3022
> Detect=HKCU\Software\Hagel\DU Meter
> FileKey1=%CommonAppData%\Hagel Technologies\DU Meter|*.csv;error.elf
> ```

**Comment by Shakil-Shahadat** [comment]
Updated.

**Comment by APMichael** [comment]
These changes are included in #824.
